hbase-client/src/main/java/org/apache/hadoop/hbase/shaded/protobuf/ProtobufUtil.java

```diff
@@ -2136,6 +2159,22 @@ public static SlowLogParams getSlowLogParams(Message message) {
     return new SlowLogParams(params);
   }
 
+  private static String toJson(
+    boolean slowLogOperationJsonEnabled,
+    int maxCols,
+    Callable<Operation> operation
+  ) {
+    if (!slowLogOperationJsonEnabled) {
+      return StringUtils.EMPTY;
+    }
+    try {
+      return operation.call().toJSON(maxCols);
+    } catch (Exception e) {
+      LOG.warn("Exception when deriving operation JSON", e);
+    }
```

Comment:
We introduce the possibility for IOEx on each `ProtobufUtil#toX` call and on `Operation#toJSON`. I made this toJson method to unify the suppression of these IOExceptions, but I'd understand if there's pushback on this approach